services/polymart/polymart-latest-version.service.js

```diff
@@ -0,0 +1,41 @@
+import { BasePolymartService, documentation } from './polymart-base.js'
+
+export default class PolymartLatestVersion extends BasePolymartService {
+  static category = 'version'
+
+  static route = {
+    base: 'polymart/version',
+    pattern: ':resourceId',
+  }
+
+  static examples = [
+    {
+      title: 'Polymart Version',
+      namedParams: {
+        resourceId: '323',
+      },
+      staticPreview: this.render({
+        version: '1.2.9',
+      }),
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'version',
+    color: 'blue',
+  }
+
+  static render({ version }) {
+    return {
+      message: `${version}`,
+    }
+  }
+
+  async handle({ resourceId }) {
+    const { response } = await this.fetch({ resourceId })
+    return this.constructor.render({
```

Comment:
Could we use `renderVersionBadge()` here?
https://github.com/badges/shields/blob/302c8606ff4c975b6310fe2cdfe58f92cede290b/lib/version.js#L144-L150

Reply:
Hello, how do you recommend adding it? It seems removed now.